data/mods/Magiclysm/Spells/attunements/Void_Mage.json

```diff
@@ -0,0 +1,44 @@
+[
+  {
+    "type": "SPELL",
+    "id": "voidmage_final_grasp",
+    "name": "Final Grasp",
+    "description": "BLANK_BLANK_BLANK_BLANK_BLANK_BLANK_BLANK_BLANK_BLANK_BLANK_BLANK_BLANK_.",
+    "effect": "attack",
+    "shape": "blast",
+    "valid_targets": [ "hostile" ],
+    "flags": [ "SILENT", "SOMATIC", "IGNORE_WALLS", "NO_LEGS", "MUST_HAVE_CLASS_TO_LEARN" ],
+    "min_damage": 0,
+    "max_damage": 0,
+    "min_range": 5,
+    "max_range": 20,
+    "range_increment": 0.4,
+    "damage_type": "pure",
+    "min_dot": 5,
+    "max_dot": 55,
+    "dot_increment": 1.25,
+    "max_level": 40,
+    "min_duration": 500,
+    "max_duration": 5000,
+    "duration_increment": 115,
+    "spell_class": "VOID_MAGE",
+    "base_casting_time": 200,
+    "energy_source": "MANA",
+    "base_energy_cost": 300,
+    "difficulty": 9,
+    "extra_effects": [ { "id": "voidmage_final_grasp_slowdown" } ]
+  },
+  {
+    "id": "voidmage_final_grasp_slowdown",
+    "type": "SPELL",
+    "name": "Final Grasp slowdown",
+    "description": "Remove some moves from enemy. Technical spell, you dont allowed to see it, if youre not in debug mode.",
+    "valid_targets": [ "hostile" ],
+    "effect": "mod_moves",
```

Comment:
This one uses mod_moves and the other one doing a slowdown uses an effect? And would this subtract -100 at the minimum? That seems rather high.

Reply:
main one make DoT effect, when second remove some move points from target - its only 1-10 moves total